### PR TITLE
perf(jest): remove ts-jest

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

- Remove ts-jest to improve speed
  - We still had this in to handle the path-alias' we created
  - Switched to manual mapping
- Remove ts-node dep
  - Switched jest config files back to js
  - Added types and usign those via comment option

### Type of change (Remove other not matching type)

- perf

### How has this been tested?

- unit tests ✅ 
- build ✅

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -15,6 +12,7 @@ module.exports = {
             },
           },
         },
+        target: "es2016",
```
Comment (escapedcat) [inline jest.config.ts:15]: Added this to avoid the weird `class` error

**Comment by escapedcat** [comment]
e2e test failing for other reasons

**Review by bumi — APPROVED** [review]
looks good. 
(from my limited experience)

**Comment by escapedcat** [comment]
> looks good. (from my limited experience)

We should just use this for all upcoming PRs :P

**Pull request merged**